libs/entitlements-runtime/src/main/java/module-info.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+module org.elasticsearch.entitlements.runtime {
+    requires org.elasticsearch.base;
+
+    exports org.elasticsearch.entitlements.runtime.api;
```

Comment:
This should be a qualified export?

Reply:
Noob question - how do I avoid the Module Not Found warning when I export this to the agent?

Reply:
Oh wait, it's another PEBCAK. Some of my modules start with `org.` and some don't.